test/acceptance/nouns.ts

```diff
@@ -55,12 +42,11 @@ export function testNounsIntegration(): void {
       });
 
       // 2. Name owner registers nouns.eth as a new guild
-      // const nounsDomain = "nouns.eth";
       await asAccount(nounsDomainOwner, async (signer) => {
         // Register guild
         await ensGuilds
           .connect(signer)
-          .registerGuild(nounsDomain, admin, flatFeePolicy.getAddress(), openAuthPolicy.getAddress());
+          .registerGuild(nounsDomain, admin, flatFeePolicy.getAddress(), nftAuthPolicy.getAddress());
```

Comment:
I think this is the config we'd recommend @gsgalloway

Reply:
Let's use allowlist auth, or a new auth that just allows no mints at all